app/src/main/res/layout-sw600dp/story_fragment.xml

```diff
@@ -36,13 +37,20 @@
         app:navigationContentDescription="@string/navigate_up"
         app:navigationIcon="?attr/homeAsUpIndicator">
 
+        <asia.ivity.android.marqueeview.MarqueeView
+          android:id="@+id/marquee_view"
```

Comment:
Since IDs are global, all IDs should be specific to their specific fragment or custom view. In this case, I suggest: ``story_toolbar_title_marquee`` ('view' is redundant). Ditto everywhere else.

Reply:
Missed this the rest are named with their corresponding Fragment e.g. exploration_marquee_view, license_text_viewer_activity_marquee_view.